charts/datadog/values.yaml

```diff
@@ -706,6 +706,12 @@ datadog:
       # datadog.sbom.containerImage.enabled -- Enable SBOM collection for container images
       enabled: false
 
+      # datadog.sbom.containerImage.uncompressedLayersSupport -- Use container runtime snapshotter
+      # This should be set to true when using EKS, GKE or if containerd is configured to
+      # discard uncompressed layers.
+      # This feature will cause the SYS_ADMIN capability to be added to the Agent container.
+      uncompressedLayersSupport: false
```

Comment:
I feel like the option name does not exactly represent what it does. Using `mount` has different facets (more performant, more permissions, etc.) and perhaps in the future we'll have another way to support uncompressed layers while still offering the `mount` option.

Reply:
This is intended: the idea of this flag is more to make the feature work in the case of discarded layers. This will allow us to more to a better solution in the future. Naming it `mountImages` would make the migration more complex as we wouldn't know if the feature was specifically request or if it's because it was used as a workaround.
Does it make sense ?

Reply:
It makes sense to have a dedicated `uncompressedLayersSupport` that activates `useMount`. What does not make sense is that `useMount` does not exist.

I know we did not advertise this option before, but I believe it should have its own existence and we could put in the docstring that `uncompressedLayersSupport` implies `useMount`, with the proper explanation on `useMount`